products/logs/backend/api.py

```diff
@@ -73,13 +73,11 @@ def attributes(self, request: Request, *args, **kwargs) -> Response:
         results = sync_execute(
             """
 SELECT
-    arrayFilter(
-        x -> x like %(search)s,
-        arraySort(groupArrayDistinctArrayMerge(attribute_keys)) as keys
-    )
+    arraySort(groupArrayDistinct(attribute_key)) as keys
 FROM log_attributes
-WHERE time_bucket >= toStartOfHour(now()) AND time_bucket <= toStartOfHour(now())
+WHERE time_bucket >= toStartOfInterval(now() - interval 1 hour, interval 10 minute)
```

Comment:
TODO: figure out how to pass the date filter ranges into the attributes lookup

Reply:
the problem is sactually getting them into this function to begin with, i couldn't follow the taxonomy filter lookup stuff